spec/form_objects/support_ticket/check_your_request_form_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe SupportTicket::CheckYourRequestForm do
+  let(:mock_ticket) do
+    {
+      'full_name' => 'Joe Blogg',
+      'email_address' => 'joe@bloggs.com',
+      'user_type' => 'parent',
+      'telephone_number' => '0207 333 4444',
+      'subject' => 'My query',
+      'message' => 'This is my query',
+      'support_topics' => %w[hello world],
+    }
+  end
+
+  describe 'when in production and znedesk creditials are provided' do
```

Comment:
```suggestion
  describe 'when Zendesk credentials are provided' do
```
Typo, and removed 'in production' as there's no mention of production in the test code